source/data/tier1data/wsl.rst

```diff
@@ -0,0 +1,51 @@
+.. _wsl:
+
+################################################
+Installing WSL2 on windows (optional)
```

Comment:
This page is not specific to tier-1 data, isn't it? It is just generic instructions to install WSL2. So I would move it out of `/data/tier1data` and remove the `optional` from the title.

A good place would be under `/access` and then we can link to it on those sections that already mention WSL such as https://docs.vscentrum.be/access/windows_client.html#windows-subsystem-for-linux-wsl-wsl2

The fact that is optional for irods should be mentioned whenever this page is linked in the tier-1 data documentation, not in the title.

Reply:
Good idea! 
Pushing soon